### DISCUSSION TITLE
how to directly publish to ring-mqtt topic?

### DISCUSSION THREAD

**Discussion by gforums** [tsightler/ring-mqtt#576] [Support Q&A] [answered]
Dear all,

I'm using this tool to unlock my intercom. Great that this was implemented that easily!

Now I have a general question and hope that you can guide me in the right direction:

- Using openhab and the home assistant addon, I can send commands succesfully to ring-mqtt.

I am failing with doing this in a manual way.
Subscribing to the topics works like a charm with mosquitto_sub but also with a simple windows mqtt tool.
Just publishing doesn't work.

I am trying to do so for the intercom lock @ 25xxxxxxxxxxxxx (ring id)/intercom/3xxxxxxxx/lock/command

I tried mosquitto_pub (-h localhost - m "UNLOCKED") and my windows tool with "UNLOCKED" / 1 / 0 / ON / OFF but nothing of them works.
Do I need to pass some more information? user name/password or whatever?
I have no user name/pw for the mqtt broker itself. 

Thanks for any hint!
Best, Christian

**Comment by tsightler** [comment, marked as answer]
As [documented](https://github.com/tsightler/ring-mqtt/wiki/MQTT-Device-Topics#locks), the command is LOCK/UNLOCK.

In general, all devices use the default payload values from the equivalent Home Assistant integration, so if I failed to document a specific devices command values in the wiki, you can refer to the HA device documentation to see what the default payload command values are.  For example, with the lock it is the Lock integration, and you can see the default payloads for all command in the [lock integration documentation](https://www.home-assistant.io/integrations/lock.mqtt/#payload_lock).

**Reply by gforums** [reply]
thanks @tsightler , indeed works as expected also for the ring intercom device. marked as answered.
